packages/nx-plugin/src/plugin/target/targets.ts

```diff
@@ -2,21 +2,26 @@ import { readdir } from 'node:fs/promises';
 import { CP_TARGET_NAME } from '../constants';
 import type { NormalizedCreateNodesContext } from '../types';
 import { createConfigurationTarget } from './configuration-target';
+import { createExecutorTarget } from './executor-target';
 
 export async function createTargets(
   normalizedContext: NormalizedCreateNodesContext,
 ) {
-  const { targetName = CP_TARGET_NAME } = normalizedContext.createOptions;
+  const { targetName = CP_TARGET_NAME, bin } = normalizedContext.createOptions;
   const rootFiles = await readdir(normalizedContext.projectRoot);
   return rootFiles.some(filename =>
     filename.match(/^code-pushup\.config.(\w*\.)*(ts|js|mjs)$/),
   )
     ? // @TODO return code-pushup cli target https://github.com/code-pushup/cli/issues/619
-      {}
+      {
+        [targetName]: createExecutorTarget({ bin }),
+      }
```

Comment:
Is the TODO comment now resolved?

Reply:
consider `projectPrefix` is missing. I added this PR in the issue to show what it resolves